sql-db/vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/handlers/MSsqlHandlerIT.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.qe.vertx.sql.handlers;
+
+import io.quarkus.test.bootstrap.DefaultService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MSsqlHandlerIT extends CommonTestCases {
+    private static final String MSSQL_PASSWORD = "QuArKuS_tEsT";
+    private static final String DATABASE = "msdb";
+    private static final int MSSQL_PORT = 1433;
+
+    @Container(image = "${mssql.image}", port = MSSQL_PORT, expectedLog = "Service Broker manager has started")
+    static DefaultService mssql = new DefaultService()
```

Comment:
It should be SqlServerService

Reply:
Why? At the moment SqlService has no onPreStart action, so all properties should be configured manually anyway, due to errors in constant values and in generating code getXUrl methods couldn't be used either. I'll fix it in framework, but until 0.0.10 there is no point in using it here.

Reply:
I was not aware of this issue. Please, submit an issue in the test framework repository and add a link here to state that will be fixed in 0.0.10 and we should change it to use SqlServerService.
Moreover, apply your workaround in the OpenShiftMssqlDatabaseIT test too.

Reply:
Done and done https://github.com/quarkus-qe/quarkus-test-framework/issues/251